geopandas/tests/test_geom_methods.py

```diff
@@ -409,6 +409,20 @@ def test_interpolate(self):
         self._test_binary_topological('interpolate', expected, self.g5,
                                       1.5)
 
+    def test_interpolate_distance_array(self):
+        expected = GeoSeries([Point(0.0, 0.75), Point(1.0, 0.5)])
+        self._test_binary_topological('interpolate', expected, self.g5,
+                                      np.array([0.75, 1.5]))
+
+        expected = GeoSeries([Point(0.5, 1.0), Point(0.0, 1.0)])
+        self._test_binary_topological('interpolate', expected, self.g5,
+                                      np.array([0.75, 1.5]), normalized=True)
+
+    def test_interpolate_distance_wrong_length(self):
+        distances = np.array([1, 2, 3])
+        with pytest.raises(ValueError):
+            self.g5.buffer(distances)
```

Comment:
This should be `simplify`, right?

Reply:
Hum good catch i guess this should be `interpolate` instead of `buffer`!